frontend/src/lib/hooks/useAddressBook/utils/isValidAddress.ts

```diff
@@ -0,0 +1,6 @@
+import WAValidator from 'multicoin-address-validator';
+
+export const isValidAddress = (address: string, coinSymbol: string) => {
+  // The 'coinSymbol' is the symbol of the cryptocurrency (e.g., 'ETH', 'BTC')
+  return WAValidator.validate(address, coinSymbol);
```

Comment:
We have the services for anything related to chains, coins and tokens. So, please use:

```
const addressService = AddressServiceFactory.createAddressService(
        chain,
        walletCore
      );
      addressService.validateAddress(address);
      ```